bedita-app/models/object_relation.php

```diff
@@ -28,6 +28,26 @@
 class ObjectRelation extends BEAppModel
 {
 
+    /**
+     * Check if a relation exists.
+     *
+     * @return bool
+     */
+    public function exists() {
```

Comment:
This method doesn't seem used. Is it correct?

Reply:
Actually this method overrides `Model::exists()` (the default CakePHP's method). It is used internally during the `Model::save()` operation to check if a new record should be created, or an existing should be updated.

It is required to override this method because by default it would only check `id`, but since we're using a composite primary key we must perform a different check.